### ISSUE TITLE
location network wrong

### ISSUE THREAD

**Issue by fatalfeel** [zhang-yi-chi/recurrent-models-of-visual-attention#2]
distribution = torch.distributions.Normal(mu, self.std)
  output = torch.clamp(distribution.sample(), -1.0, 1.0)
  log_p = distribution.log_prob(output)
  log_p = torch.sum(log_p, dim=1)

/////////////should be correct
      distribution    = torch.distributions.Normal(mu, self.location_std)
        lt              = distribution.sample() #should not clamp first
        log_p           = distribution.log_prob(lt)
        log_p           = torch.sum(log_p, dim=1)
        lt              = torch.clamp(lt, -1, 1)
